{"uuid":"9a69ce61-a743-4153-bd92-6259a8eb69dd","historyId":"ce926dd725fa848292bc8fb6bc0caa7d","fullName":"Array: The user is presented with error message for code with invalid syntax in tryEditor for Arrays in Python page","labels":[{"name":"tag","value":"arrays"},{"name":"tag","value":"scenario_016"},{"name":"host","value":"DESKTOP-RGL4EEL"},{"name":"thread","value":"19632@DESKTOP-RGL4EEL.main(1)"},{"name":"feature","value":"Array"},{"name":"story","value":"The user is presented with error message for code with invalid syntax in tryEditor for Arrays in Python page"},{"name":"suite","value":"Array"},{"name":"testClass","value":"The user is presented with error message for code with invalid syntax in tryEditor for Arrays in Python page"},{"name":"framework","value":"cucumber4jvm"},{"name":"language","value":"java"},{"name":"gherkin_uri","value":"file:///C:/Users/janga/eclipse-workspace/DSAlgosample/src/test/resources/Features/UC05_Arrays.feature"},{"name":"package","value":"src.test.resources.Features.UC05_Arrays_feature.Array"}],"links":[],"name":"The user is presented with error message for code with invalid syntax in tryEditor for Arrays in Python page","status":"passed","stage":"finished","steps":[{"name":"Given  The user is in a page having an tryEditor with a Run button to test","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1680119002191,"stop":1680119002312},{"name":"When  The user enter python code with invalid syntax in tryEditor from sheet \"pythonCode\" and 1","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1680119002312,"stop":1680119002622},{"name":"And  The user clicks on run button","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1680119002623,"stop":1680119002797},{"name":"Then  The user should be presented with error message as \"NameError: name 'hello' is not defined on line 1\"","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1680119002798,"stop":1680119002805}],"attachments":[],"parameters":[{"name":"Sheetname","value":"pythonCode"},{"name":"RowNumber","value":"1"}],"start":1680119002190,"stop":1680119002806}